contest.html

```diff
@@ -0,0 +1,93 @@
+
+               <a href="vote.html" style="padding-left:5px; "><div class="icon-box" style="padding-left: 8px; width: 43px !important;" ><span class="side-icon" style="margin-right: 10px !important;"><i class="fas fa-vote-yea"></i></span></div> <p>VOTE</p></a> 
+               
+               <a href="contest.html" style="padding-left:5px "><div class="icon-box" style="padding-left: 8px; width: 43px !important;" ><span class="side-icon" style="margin-right: 10px !important;"><i class="fas fa-file-signature"></i></span></div> <p>CONTEST</p></a>
+          </div>
+          <div class="main-box">
+              <div class="main-box-inner party-box">
+
+                <h1>Contest Form</h1>
+                <hr>
+                
+                
+
+                <div class="card-row">
+                        <form id="contest-form" action="" style="border:1px solid #ccc">
+                            <div class="contain">
+                              
+                              <label for="firstname"><b>Firstname</b></label>
+                              <input id="fN" type="text" placeholder=" " name="firstname" required>
+                              <label for="lastname"><b>Lastname</b></label>
+                              <input id="lN" type="text" placeholder="" name="last" required>
+                              <label for="party"><b>Party</b></label>
+                              <input id="pty" type="text" placeholder="" name="party" required>
+                              <label for="office"><b>Office</b></label>
+                              <input id="of" type="text" placeholder="" name="office" required>
+                              
+                              <label for="manifesto"><b>Manifesto</b></label>
+                              <textarea id="man" name="manifesto" rows="10" cols="100" required></textarea>
+                          
+                              <div class="clearfix">
+                                <button type="button" class="cancelbtn" onclick="clearForm()">CANCEL</button>
+                                <button type="submit" class="signupbtn" onclick="conSubmit()">SUBMIT</button>
+                              </div>
+                            </div>
+                          </form>
+                </div>
+                      
+                
+                         
+                
+              </div>
+              
+          </div>
+
+      <div id="footer-main" class="footer">
+            <div class="wrapper">
+                <div class="footer-main">
+                      <p>Copyright &copy; 2019 Politico</p>
+                      <p>This is the beta version of a test product of ICE Inc.</p>
+        
+                </div>
+           </div>
+      </div> 
+   
+   <script src="js/script.js"></script>   
+  </body>
+
+  </html>
```

Comment:
A new line here, please

Reply:
Thanks for pointing that out. I will effect the change ASAP.